fvm/src/gas/price_list.rs

```diff
@@ -124,85 +131,91 @@ lazy_static! {
         get_randomness_per_byte: 0,
 
         block_memcpy_per_byte_cost: 0,
-        block_io_per_byte_cost: 0,
-        block_link_per_byte_cost: 1,
 
-        block_open_base: 114617,
-        block_read_base: 0,
+        block_open_base: static_milligas!(114617),
+        block_open_memret_per_byte_cost: 0,
```

Comment:
What is memret?

Reply:
The memory retention cost (see section under https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0032.md#ipld-state-management-gas).

Reply:
(Can rename to something else, if it sounds like memory return).